pkgs/data/fonts/oxygenfonts/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, fetchFromGitHub }:
+
+let
+  pname = "oxygenfonts";
+  version = "unstable-20140213";
+in
+
+  stdenv.mkDerivation rec {
+    name = "${pname}-${version}";
+
+    src = fetchFromGitHub {
+      owner = "vernnobile";
+      repo = "oxygenFont";
+      rev = "62db0ebe3488c936406685485071a54e3d18473b";
+      sha256 = "134kx3d0g3zdkw8kl8p6j37fzw3bl163jv2dx4dk1451f3ramcnh";
+    };
+
+    phases = [ "unpackPhase" "installPhase" ];
+
+    installPhase = ''
+      mkdir -p $out/share/fonts/${pname}
+      cp */*.ttf $out/share/fonts/${pname}
```

Comment:
Looking at the downloaded archive I feel a bit confused since it contains a few different places where TTF files are present:
```
rycee@beta:~$ find /nix/store/7kb5nibz3zyn172m3fcvlbbrqx5b9mpc-oxygenFont-62db0ebe3488c936406685485071a54e3d18473b-src -name *.ttf
/nix/store/7kb…mpc-oxygenFont-62d…73b-src/Oxygen-GoogleWebFont/Oxygen-Bold.ttf
/nix/store/7kb…mpc-oxygenFont-62d…73b-src/Oxygen-GoogleWebFont/Oxygen-Light.ttf
/nix/store/7kb…mpc-oxygenFont-62d…73b-src/Oxygen-GoogleWebFont/Oxygen-Regular.ttf
/nix/store/7kb…mpc-oxygenFont-62d…73b-src/Oxygen-GoogleWebFont/src/Oxygen-Bold-unhinted.ttf
/nix/store/7kb…mpc-oxygenFont-62d…73b-src/Oxygen-GoogleWebFont/src/Oxygen-Light-unhinted.ttf
/nix/store/7kb…mpc-oxygenFont-62d…73b-src/Oxygen-GoogleWebFont/src/Oxygen-Regular-unhinted.ttf
/nix/store/7kb…mpc-oxygenFont-62d…73b-src/Oxygen-Monospace/OxygenMono-Regular.ttf
/nix/store/7kb…mpc-oxygenFont-62d…73b-src/Oxygen-Monospace/src/OxygenMono-Regular-unhinted.ttf
/nix/store/7kb…mpc-oxygenFont-62d…73b-src/OxygenSans-version-0.2/Oxygen-Bold/Oxygen-Bold.ttf
/nix/store/7kb…mpc-oxygenFont-62d…73b-src/OxygenSans-version-0.2/Oxygen-Bold/src/Oxygen-Bold-unhinted.ttf
/nix/store/7kb…mpc-oxygenFont-62d…73b-src/OxygenSans-version-0.2/Oxygen-Regular/Oxygen.ttf
/nix/store/7kb…mpc-oxygenFont-62d…73b-src/OxygenSans-version-0.2/Oxygen-Regular/src/Oxygen-unhinted.ttf
/nix/store/7kb…mpc-oxygenFont-62d…73b-src/OxygenSans-version-0.4/Bold-700/Oxygen-Sans-Bold.ttf
/nix/store/7kb…mpc-oxygenFont-62d…73b-src/OxygenSans-version-0.4/Bold-700/src/Oxygen-Sans-Bold-unhinted.ttf
/nix/store/7kb…mpc-oxygenFont-62d…73b-src/OxygenSans-version-0.4/Regular-400/Oxygen-Sans.ttf
/nix/store/7kb…mpc-oxygenFont-62d…73b-src/OxygenSans-version-0.4/Regular-400/src/Oxygen-Sans-unhinted.ttf
```
So the `cp` line in the installation phase above will copy the TTFs from the Google web fonts directory and the mono directory. Is this OK?